pyproject.toml

```diff
@@ -40,6 +40,7 @@ dependencies = [
   "frequenz-api-dispatch == 1.0.0-rc1",
   "frequenz-client-base >= 0.8.0, < 0.10.0",
   "frequenz-client-common >= 0.1.0, < 0.4.0",
+  "frequenz-client-microgrid >= 0.7.0, < 0.8.0",
```

Comment:
Let's not do this yet. If needed we should move it to `frequenz-client-common`. I will do a PR soon.

Reply:
But I need this now to move actor to latest SDK :)

Reply:
TargetComponents is part of DispatchRequest.
Actor receives DispatchRequest and needs to create BatteryPool from the target.
BatteryPool expects set[ComponentId] :) 

I can to change every actor to change `int to ComponentId.
But after we add this, I would have to revert this change in every actor.

Reply:
Mmmm, damn. I guess then I'm back to the "OK as a temporary thing" comment :grimacing:

Reply:
> please create an issue so we don't forget to remove the dependency in the future.

https://github.com/frequenz-floss/frequenz-client-dispatch-python/issues/172

Reply:
We can use typing.SupportsInt and support the unknown to int castable ComponentsId that way

Reply:
how it would fix it? The problem with Actor will still be there. They except ComponentId, get int.